kivy/uix/recycleview/views.py

```diff
@@ -108,6 +111,34 @@ def apply_selection(self, rv, index, is_selected):
         pass
 
 
+class RecycleKVIDsDataViewBehavior(RecycleDataViewBehavior):
+    """Similar to :class:`RecycleDataViewBehavior`, except that the data keys
+    can signify properties of objects named with an id in KV.
+
+    E.g. given a KV rule::
+
+        <MyRule@RecycleKVIDsDataViewBehavior+BoxLayout>:
+            Label:
+                id: name
+            Label:
+                id: value
+
+    Then setting the data list with
+    ``rv.data = [{'name.text': 'Kivy user', 'value.text': '12'}]`` would
+    automatically set the corresponding labels.
```

Comment:
a versionadded flag for the class would be nice.

Reply:
Done